tests/fixtures/gtfs2ntfs/no_traffic/input/stops.txt

```diff
@@ -1,5 +1,5 @@
-stop_id,stop_name,stop_lat,stop_lon,location_type,parent_station
-stoparea:1,plop,48.844746,2.372987,1,
-stop:31,pouet,48.844746,2.372987,0,stoparea:1
-stop:32,pouet,48.844746,2.372987,0,stoparea:1
-stop:33,pouet,48.844746,2.372987,0,stoparea:1
+stop_id,stop_name,stop_lat,stop_lon,location_type,parent_station,level_id
```

Comment:
Maybe add files levels.txt in /input and /output. Otherwise gtfs and ntfs are not valid as well (orphan keys level_id).